src/TestSuite/Fixture/FixtureInjector.php

```diff
@@ -14,6 +14,21 @@
  */
 namespace Cake\TestSuite\Fixture;
 
+if (class_exists('PHPUnit_Runner_Version')) {
+    if (version_compare(\PHPUnit_Runner_Version::id(), '5.7', '<')) {
+        trigger_error(sprintf('Your PHPUnit Version must be at least 5.7.0 to use CakePHP Testsuite, found %s', \PHPUnit_Runner_Version::id()), E_USER_ERROR);
+    }
+    class_alias('PHPUnit_Framework_Test', 'PHPUnit\Framework\Test');
+    class_alias('PHPUnit_Framework_Warning', 'PHPUnit\Framework\Warning');
```

Comment:
Using the `PHPUnit_Framework_Version` class also determines if PHPUnit <6 is present at all and only defines those aliases in this case